lldb/include/lldb/Target/ThreadPlan.h

```diff
@@ -590,6 +591,7 @@ class ThreadPlanNull : public ThreadPlan {
   const Status &GetStatus() { return m_status; }
 
 protected:
+  friend class ThreadPlanSingleThreadTimeout;
```

Comment:
What is this needed for?  It's a little odd having a derived class friended to the base class, since you could make whatever the derived class needed protected rather than private.

Reply:
Hmm, I thought there is a reason I can't remember, apparently not. Removed. Thanks.